digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/service/linearasset/LinearAssetService.scala

```diff
@@ -157,23 +150,67 @@ trait LinearAssetOperations {
     (assetId, middePoint)
   }
 
+  def getLinearMiddlePointAndSourceById(typeId: Int, assetId: Long): (Long, Option[Point], Option[Int])  = {
```

Comment:
this method replace 'getLinearMiddlePointById'
getLinearMiddlePointById method is no longer used, please delete